silver/tests/unit/test_pdf.py

```diff
@@ -86,14 +86,14 @@ def test_pdf_mark_as_clean_mimic_2_threads():
 
 
 @pytest.mark.django_db
-def test_pdf_mark_as_dirty_min_value():
-    # somehow inconsistent state
-    pdf = PDF.objects.create(dirty=-1)
+def test_pdf_mark_as_clean_min_value():
```

Comment:
Why did this test have to be changed?

Reply:
The main purpose to fit to the new feature in Django 2.2 — extended constraints.
With 2.2 you can't apply `pdf = PDF.objects.create(dirty=-1)` because integrity error will be produced (`dirty` is PositiveIntegerField).

As I understand `test_pdf_mark_as_dirty_min_value` was written to be sure that `dirty` can not be below 0. So it seems like with the new 2.2 feature and also with you changes (https://github.com/silverapp/silver/commit/e4dd96ef44bf62f52a7f939a213af4e6bad1ed6a) the test_pdf_mark_as_clean_min_value is unnecessary, test_pdf_mark_as_clean_min_value is enough.

Reply:
Makes sense. Thanks!